### DISCUSSION TITLE
Supabase Auth Vanity Values

### DISCUSSION THREAD

**Discussion by Steellgold** [supabase/supabase#13575] [Feature Requests]
When we want to define values for our users, we have to create our own table, I propose that in the Auth part we can define new tables that we can modify with a function on our site for example

Exemple:

`[...].edit("superValue", 20)`
`[...].get("superValue")`

![image](https://user-images.githubusercontent.com/51505384/230898513-7eb048e9-7533-49a2-b4a9-1d0dbf957c9b.png)

**Comment by NanoCode012** [comment]
I think custom claims can allow you to set values: https://github.com/supabase-community/supabase-custom-claims
